can/interfaces/pcan/pcan.py

```diff
@@ -86,9 +86,15 @@ def __init__(self, channel='PCAN_USBBUS1', state=BusState.ACTIVE, bitrate=500000
             Bitrate of channel in bit/s.
```

Comment:
We should document that this may not only be the bitrate of the channel, but also possibly the SJW, BRP, TSEG2 and TSEG1.

Reply:
I added it in the doc string

Reply:
Nice!